features/backend/product_archetypes.feature

```diff
@@ -76,6 +76,7 @@ Feature: Product archetypes
         Then I should be on the product archetype index page
         And I should see "Archetype has been successfully created."
 
+    @javascript
     Scenario: Creating a product by building archetype
```

Comment:
We could, but *de facto*, it will be a little bit duplication... Nevertheless this scenario checks creating product based on archetype, so we don't have to check it in the other one.